llvm/test/Transforms/Util/loop-metadata.ll

```diff
@@ -402,6 +402,78 @@ for.body:                                         ; preds = %for.cond
 }
 
 
+; find IV even if it is hidden in a truncation operation
```

Comment:
I don't see `trunc` in those tests 😕

Reply:
curretly checking if i really need to process the truncated SCEV